### DISCUSSION TITLE
Showing a fatal error when component view is not wrapped on a <div>

### DISCUSSION THREAD

**Discussion by ppastene** [livewire/livewire#1322] [4. General]
Hello

I'm learning Livewire since I'm starting to learn about frontend frameworks like ReactJs and VueJS. I've reached Livewire and I've found that is a promising framework to use it in Laravel. I've written some components and I've done some newbie mistakes that took me a lot of time figuring out like printing properties outside of the <div> tag, so as a suggestion it could be nice to trigger a warning just like in Lavarel if there are error syntax like a not closed <div> or elements outside of them just like in ReactJS and VueJS

**Comment by andradei** [comment]
I'm with you onthis. I spent hours trying to find out why my nested component wasn't being sent to the correct route.

When I interacted with it, it was sending AJAX requests to the parent component's route.

It only worked when I wrapped it in a `<div>`.

I'm still enjoying learning livewire a ton.

UPDATE:

I dug the issues, other discussions, and documentation and found out that at some point (probably 1.0) the `<div>` root element was required. Now it can be other elements, but `<div>` is still preferred to the point `php artisan livewire:make` creates a component with

```html
<div>
    {{-- Some cool/wise quote --}}
</div>
```
